_i18n/nl.yml

```diff
@@ -0,0 +1,314 @@
+  chainlocks: Chainlocks  (gedaan)
+  chainlocksdesc: Chainlocks maken gebruik van Firo's masternodenetwerk om bescherming te bieden tegen 51%-aanvallen. Transacties worden definitief zodra 1 blok is bevestigd.
+  rap: RAP-adressen  (gedaan)
+  rapdesc: RAP-adressen staan een adres toe dat publiekelijk kan worden gedeeld, zodat derden niet kunnen zien hoeveel of wanneer het geld is ontvangen.
+  instant: Instant Send  (gedaan)
+  instantdesc: Instant Send maakt gebruik van Firo's masternodenetwerk om transacties binnen een paar seconden te bevestigen, waardoor het praktisch kan worden gebruikt als ruilmiddel.
+  elysium: Elysium (on Testnet)
+  elysiumdesc: Elysium maakt het mogelijk om tokens (bijvoorbeeld stablecoins, asset backed coins) uit te geven op het netwerk van Firo die profiteren van de privacytechnologie van Firo.
+  elysiumv2: Elysium v2
+  elysiumv2desc: Elysium v2 breidt de mogelijkheden van Elysium uit met eenvoudigere bridging en DeFi mogelijkheden. Elysium v2 zal ook Spats implementeren die het type activa verbergen dat wordt overgedragen.
+  lelantusspark: Lelantus Spark (gedaan)
+  lelantussparkdesc: Lelantus Spark maakt directe anonieme betalingen mogelijk waardoor gebruikers met elkaar kunnen handelen zonder bedragen, bron en ontvanger te onthullen. Spark introduceert een Spark-adressen waarmee gebruikers hun geld volledig verborgen kunnen houden, samen met volledige ondersteuning voor view key en threshold multisig.
+  mobile: Mobile Wallet  (gedaan)
+  mobiledesc: Een native, standaard private mobiele portemonnee met volledige ondersteuning voor Lelantus-transacties.
+  firopow: FiroPoW  (gedaan)
+  firopowdesc: FiroPoW is een ProgPOW-variant die is ontworpen voor GPU's en sterk bestand is tegen ASIC's.
+  research: Research
+  lelantussparkaddress: Lelantus Spark sub-addressing and payment proofs (gedaan)
+  lelantussparkaddressdesc: Lelantus Spark ondersteunt subadressen die efficiënter scannen mogelijk maken en betalingsbewijzen waarmee gebruikers kunnen bewijzen dat ze een betaling hebben gedaan.
+  firodex: FiroDEX  (gedaan)
+  firodexdesc: FiroDEX biedt gebruikers de mogelijkheid om van en naar Firo te swappen met behulp van een gedecentraliseerde infrastructuur die gebruik maakt van atomic swap-technologie.
+  bridges: Cross-chain bridges
+  bridgesdesc: Cross chain bridges zullen tokens van andere ecosystemen in en uit Firo's Elysium-tokenlaag laten overbruggen om gebruik te maken van onze privacy-infrastructuur.
+  mixnets: Mixnets
+  mixnetsdesc: Het tunnelen van transacties en knooppuntcommunicatie in mixnets biedt een betere bescherming van metadata, zelfs tegen tegenstanders op staatsniveau. We onderzoeken het Meson-mixnet dat is gebouwd op Katzenpost-softwarebibliotheken.

```

Comment:
```suggestion
  mixnetsdesc: Het tunnelen van transacties en knooppunt communicatie in mixnets biedt een betere bescherming van metadata, zelfs tegen tegenstanders op staatsniveau. We onderzoeken het Meson-mixnet dat is gebouwd op Katzenpost-softwarebibliotheken.

```

Reply:
Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.